bellhop/templates/login/home.html.hbs

```diff
@@ -3,12 +3,16 @@
 {{/inline}}
 
 {{#*inline "content"}}
-    <form class="pure-form" method="post">
-        Email:<br>
-        <input  type="text" name="email">
-        <br>
-        <br>
-        <input class="pure-button custom-button" type="submit" value="Submit">
+    <form class="pure-form pure-form-aligned" method="post">
+        <fieldset>
+            <div class="pure-control-group">
+                <label for="email">Email:</label>
+                <input type="text" name="email" value="ew@example.com">
```

Comment:
Unless you're setting this to help the demo site, I suggest `placeholder` instead. e.g. https://purecss.io/forms/

Reply:
The demo site is pretty much exactly what this is for.